src/index.jsx

```diff
@@ -20,81 +126,101 @@ function init(App) {
    * - the compiled elm app
    * - the app bindings to the native ui
   */ 
+
+  const app = {
```

Comment:
I thought this would be a nicer way of structuring globals for our `customElementScript`